### PR TITLE
Feature/feedback adaptive

### PR DESCRIPTION
**Pull request by sivlvy**

### CODE REVIEW

File: src/components/feedbacks/Feedbacks.module.scss [hunk 1]
```diff
@@ -204,4 +204,71 @@ color: $green-dark-200;
...
+    width: 100%;
+  }
+
+}
+
+
+@media screen and (max-width: 768px) {
+  //.formContainer {
+  //  display: none;
+  //}
```
Comment (pqoqubbw) [inline src/components/feedbacks/Feedbacks.module.scss]: remove unused code

**Review by pqoqubbw — CHANGES_REQUESTED** [review]

**Comment by sivlvy** [comment]
https://app.clickup.com/t/869563kng

**Review by pqoqubbw — APPROVED** [review]

File: src/components/feedbacks/form/FeedbackForm.tsx [hunk 2]
```diff
@@ -79,7 +80,11 @@ const FeedbackForm = () => {
       <h3 className={styles.subtitle}>Залишити відгук</h3>
       {isLoggedIn ? (
         <form className={styles.feedbackForm} onSubmit={handleSubmit}>
-          <input type="text" className={styles.feedbackFormInput} />
+          <input
```
Comment (pqoqubbw) [inline src/components/feedbacks/form/FeedbackForm.tsx]: use input from mui

**Review by pqoqubbw — CHANGES_REQUESTED** [review]

**Review by pqoqubbw — APPROVED** [review]

File: src/components/Layout/header/header-nav/headerNav.tsx [hunk 3]
```diff
@@ -13,7 +13,7 @@ const HeaderNav = () => {
   const allCategories = useSelector(
     (state: RootState) => state.items.allCategories
   )
-
+  console.log(allCategories)
```
Comment (pqoqubbw) [inline src/components/Layout/header/header-nav/headerNav.tsx]: remove console.log

File: src/redux/products/operation.ts [hunk 4]
```diff
@@ -67,7 +67,8 @@ export const fetchAllCategories = createAsyncThunk<
   FetchAllCategoriesParams
 >('api/allCategories', async ({ operationType }, thunkAPI) => {
   try {
-    const response = await axiosInstance.get('api/product_category/all')
+    const response = await axiosInstance.get('/api/product_category/all')
+    console.log(response.data)
```
Comment (pqoqubbw) [inline src/redux/products/operation.ts]: same here

**Review by pqoqubbw — CHANGES_REQUESTED** [review]

**Pull request merged**